packages/siopv2-oid4vp-op-auth/src/localization/Localization.ts

```diff
@@ -0,0 +1,49 @@
+import i18n, { Scope, TranslateOptions } from 'i18n-js'
+import memoize from 'lodash.memoize'
+import { SupportedLanguage } from '../types'
+
+class Localization {
```

Comment:
Hmzz, probably this should move to core, or better yet an i18n plugin with slight adjustments to enable other SDK modules to define there translation locations, so that they can get an instance per module with its own i18n

Reply:
PR merged